libstorage/storage.h

```diff
@@ -14,23 +14,20 @@
 #ifndef _STORAGE_H_
 #define _STORAGE_H_
 
+#include "fs.h"
+#include "dev.h"
```

Comment:
Putting headers to be installed in `include` dir might a good idea.

Reply:
Done